extensions/ql-vscode/src/model-editor/supported-languages.ts

```diff
@@ -9,20 +9,16 @@ export const SUPPORTED_LANGUAGES: QueryLanguage[] = [
   QueryLanguage.Java,
   QueryLanguage.CSharp,
   QueryLanguage.Ruby,
+  QueryLanguage.Python,
 ];
 
 export function isSupportedLanguage(
   language: QueryLanguage,
-  modelConfig: ModelConfig,
+  _modelConfig: ModelConfig,
```

Comment:
I've left this parameter here for now since it's likely we'll be adding new languages at some point in the future with a feature flag.